dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -223,7 +224,7 @@ private void addCustomMappingForRelationships(final String indexName,
                                             + "\"ignore_above\": 8191\n"
                                             + "}")));
                     mappingAPI.putMapping(indexName, properties.toString());
-                } catch (Exception e) {
+                } catch (JSONException | IOException e) {
```

Comment:
This change is really dangerous because can break a full reindex just because a field cannot be reindexed

Reply:
@wezell please take a look

Reply:
@nollymar thanks for bringing this to my attention - there are a number of unchecked exceptions that could be thrown here and we need to catch them all.  Do not merge this